script.js

```diff
@@ -0,0 +1,58 @@
+// dom - selectors
+const rpsBtns = document.querySelectorAll('.rps-btn');
+const chooseContainer = document.querySelector('.choose-container');
+const matchContainer = document.querySelector('.match-container');
+const userSelectionDisplay = document.querySelector('.user-choise');
+// dom - create elements
+const createButton = document.createElement('button');
+const createImg = document.createElement('img');
```

Comment:
In here you are creating an element but you are not using it until some of your rps buttons functions are being called. It can take some time and In my opinion, you can create those elements only when the function is being called. By doing so you are not creating 'createImg' and 'createButton' when they are not needed.

Reply:
Ok, I will create and use them only when needed.